.github/workflows/terraform-plan-apply-manual.yml

```diff
@@ -162,13 +162,13 @@ jobs:
                          -var="site_image_name=${{ matrix.site_image_name }}" \
                          -var="environment=$ENV" \
                          -var="practitioner_image_name=${{ matrix.practitioner_image_name }}" \
-                         -var="trial_config_service_image_name=${{ matrix.trial_config_service_image_name }}" \
+                         -var="trial_init_service_image_name=${{ matrix.trial_init_service_image_name }}" \
                          -var="trial_sc_gateway_image_name=${{ matrix.trial_sc_gateway_image_name }}" \
                          -var="trial_sc_discovery_image_name=${{ matrix.trial_sc_discovery_image_name }}" \
                          -var="trial_sc_config_image_name=${{ matrix.trial_sc_config_image_name }}" \
                          -var="site_image_tag=${{ matrix.site_image_tag }}" \
                          -var="practitioner_image_tag=${{ matrix.practitioner_image_tag }}" \
-                         -var="trial_config_service_image_tag=${{ matrix.trial_config_service_image_tag }}" \
+                         -var="trial_init_service_image_tag=${{ matrix.trial_init_service_image_tag }}" \
                          -var="trial_sc_gateway_image_tag=${{ matrix.trial_sc_gateway_image_tag }}" \
                          -var="trial_sc_discovery_image_tag=${{ matrix.trial_sc_discovery_image_tag }}" \
                          -var="trial_sc_config_image_tag=${{ matrix.trial_sc_config_image_tag }}" \
```

Comment:
why some have a "trial" prefix and some don't?

Reply:
probably typo. will remove the trial prefix completely.